src/lib/Libwin/passwd.c

```diff
@@ -4139,8 +3879,8 @@ wsystem(char *cmdline, HANDLE user_handle)
 	char			current_dir[MAX_PATH+1] = {'\0'};
 	char			*temp_dir = NULL;
 	int			changed_dir = 0;
-	DWORD		stat;
-	LPVOID  user_env = NULL;
+	DWORD			stat;
```

Comment:
can we please format all the local variables to be aligned? Using spaces instead of tabs is the best idea!!